tests/inside_worker_test/cast_to_float_or_null_test.py

```diff
@@ -0,0 +1,42 @@
+import pytest
+import sqlalchemy
+
+from tests.inside_worker_test.conftest import slow
+
+
+@pytest.fixture(params=[2, 2.2, 3.898986, "3.898986", "6", "0.2", 0.6])
```

Comment:
Examples with negative numbers and with scientific notation wouldn't hurt. Not sure why to have both floats and strings here, as you convert them to strings for the test anyway (and to floats for the assertion).